x/stake/types/errors.go

```diff
@@ -44,6 +44,10 @@ func ErrValidatorPubKeyExists(codespace sdk.CodespaceType) sdk.Error {
 	return sdk.NewError(codespace, CodeInvalidValidator, "validator already exist for this pubkey, must use new validator pubkey")
 }
 
+func ErrValidatorPubKeyTypeNotSupported(codespace sdk.CodespaceType) sdk.Error {
```

Comment:
Can we maybe add the type as a parameter? Indicating what exact type is unsupported.